fr.openfisca.org/fr.openfisca.org.conf

```diff
@@ -57,6 +57,14 @@ server {
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     }
 
+    location /api/v22 {
+        # new url versioning shema
+        proxy_pass http://localhost:8022/;
+        proxy_set_header Host $http_host/api/v22;
+        proxy_http_version 1.1;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    }
+
```

Comment:
Needs to be rebased on master and to add `v22` to the list of served versions. 
Feel free to force-push 🙂